### PR TITLE
Add ability to use config.toml or config dir to set templates/schemes URLs

### PR DESCRIPTION
**Pull request by jocelynthode**
Fixes https://github.com/Misterio77/flavours/issues/53

As this is my first ever use of Rust, I'm open to any feedback on the code.

Cheers

### CODE REVIEW

**Comment by jocelynthode** [comment]
@Misterio77 hey, did you have time to review this pr yet?

**Comment by Misterio77** [comment]
Hey, sorry for the wait!

I'm on my semester finals right now and things have been a little crazy. This PR is quite importante, so just a few more days

**Comment by jocelynthode** [comment]
No worries! I remember quite well  back when I had my finals as well. Take all the time you need and Good Luck!

**Comment by jocelynthode** [comment]
Hey Any news on this pr?

**Comment by Misterio77** [comment]
Sorry for the wait, getting through my backlog now!

File: src/find.rs [hunk 1]
```diff
@@ -61,3 +61,24 @@ pub fn find_template(
...
+        Ok(sources_config_file)
+    } else if sources_data_file.is_file() {
+        Ok(sources_data_file)
+    } else {
+        panic!(
+            "Neither {:?} or {:?} exist",
+            sources_config_file, sources_data_file
+        )
+    }
+}
```
Comment (Misterio77) [inline src/find.rs:84]: I can't seem to find where this is function is called from. Dead code?

**Comment by Misterio77** [comment]
Superseded by https://github.com/Misterio77/flavours/pull/62, I think?

Feel free to reopen if there's anything missing.

**Pull request closed without merging**